src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -62,10 +71,11 @@ public void setAll(double drumSpeed, double feedWheelSpeed) {
    */
   public void setDrum(double speed) {
     m_drumMotor.set(speed);
-    if (m_pdp.getCurrent(RobotMap.PDP.DRUM_PORT) > Constants.REVOLVER_DRUM_CURRENT_THRESHOLD) {
-      m_repsAboveDrum++;
-    } else {
-      resetDrumReps();
+    if (m_drumEncoder.getStopped()){
+      m_drumJammed = true;
+    }
+    else{
+      m_drumJammed = false;
```

Comment:
Instead of updating boolean flags, it makes more sense to take some kind of action here to unjam the revolver